staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -34,8 +34,14 @@ type CustomResourceDefinitionSpec struct {
 	// Scope indicates whether this resource is cluster or namespace scoped.  Default is namespaced
 	Scope ResourceScope
 	// Validation describes the validation methods for CustomResources
+	// Optional, the global validation schema for all versions. Each version can override the validation
```

Comment:
So far, I prefer how it was described in the KEP. Still trying to understand the reasons for making things overrides.